### PR TITLE
8987654: test fix

### PR DESCRIPTION
**Pull request by naotoj**
Testing. Please review 😃 
/reviewer add @LanceAndersen, @JoeWang-Java

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Issue
 * ⚠️ Failed to retrieve information on issue `8987654`.

### Reviewers
 * Lance Andersen ([lancea](@LanceAndersen) - **Reviewer**) ⚠️ Review applies to 5a8d148a5d58009f652c56ac567530bc868bfbdd
 * Joe Wang ([joehw](@JoeWang-Java) - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/playground pull/42/head:pull/42`
`$ git checkout pull/42`

### CODE REVIEW

**Comment by naotoj** [comment]
> Amazing work devleoping on this difficult fix Naoto.
> 
> Looks good. Thank you for your efforts here :-)

Thanks, yeah it was tough :-)

**Comment by JoeWang-Java** [comment]
Wow, you guys are so awesome!

**Comment by JoeWang-Java** [comment]
Commented too early -- before the bot was able to finish creating the Webrevs ;-)

**Comment by naotoj** [comment]
Yep, webrev bot is a bit slow. Probably needs some Java coffee.

File: test/jdk/sun/text/resources/removeme [hunk 1]
```diff
@@ -0,0 +1 @@
+remove me.
```
Comment (JoeWang-Java) [inline test/jdk/sun/text/resources/removeme:1]: test Add single comment

Comment (JoeWang-Java) [reply test/jdk/sun/text/resources/removeme:1]: Test "Start a review"

**Comment by JoeWang-Java** [comment]
The current webrev is: pr=42&range=00, I wonder if it'd increment to something like pr=42&range=01 ;-)

**Comment by naotoj** [comment]
Hmm, I just pushed the modified fix, but webrev bot does not seem to be working. Or is it just slow?

**Comment by LanceAndersen** [comment]
Slow I am ;-)

**Comment by JoeWang-Java** [comment]
Right, no new webrev (as Joe promised :-))
it does list commits in a dropdown box:
"show all changes": 
     + I said "remove me."

"show changes since your last review: 
      - remove me.
      + I said "remove me."
    
and also by the name of the original and new commits

**Comment by naotoj** [comment]
Thank you, guys! I'd appreciate either of you remove the removeme file. I'd be happy to review :-)

**Comment by JoeWang-Java** [comment]
wait, it (the bot) was just slow, and it's indeed range=01!

**Comment by naotoj** [comment]
> wait, it (the bot) was just slow, and it's indeed range=01!

Ah!

**Pull request closed without merging**